frontend/javascripts/oxalis/model_initialization.js

```diff
@@ -122,7 +122,13 @@ export async function initialize(
   // There is no need to reinstantiate the DataLayers if the dataset didn't change.
   if (initialFetch) {
     initializationInformation = initializeDataLayerInstances();
-    if (tracing != null) Store.dispatch(setZoomStepAction(getSomeServerTracing(tracing).zoomLevel));
+    if (tracing != null) {
```

Comment:
I'd suggest not moving the code here, but rather to keep it in the `determineDefaultState` function, because you would expect it there. This line just sets "any" zoomstep, because other steps early in the initialization need a valid zoomstep.
I'll add a suggestion in the `determineDefaultState` function.

Reply:
So just to make it clear, I'd revert the changes in this file and apply the suggested fix in the `model_initialization.js`